python/lsst/ctrl/provenance/dc3.py

```diff
@@ -155,7 +159,7 @@ def queryRunOffset(self):
         self._globalDb.condParamString("runId", self._runid)
         self._globalDb.setQueryWhere("runId = :runId")
         self._globalDb.query()
-        if not self._globalDb.next() or self._globalDb.columnIsNull(0):
+        if not next(self._globalDb) or self._globalDb.columnIsNull(0):
```

Comment:
This might be able to throw a `StopIteration` exception but I'm not entirely sure what `daf_persistence` does with `DbStorage` when it runs out of items.